frontend/test/metabase/scenarios/admin/settings/localization.cy.spec.js

```diff
@@ -1,4 +1,7 @@
-import { restore, withSampleDataset, signInAsAdmin } from "__support__/cypress";
+import { restore, signInAsAdmin } from "__support__/cypress";
+import { SAMPLE_DATASET } from "__support__/cypress_sample_dataset";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATASET;
```

Comment:
🥳

Reply:
Look at that! Access to the ids without nesting everything inside a callback! 🎉

Reply:
@paulrosenzweig you mean [THIS](https://github.com/metabase/metabase/pull/14004) 🤓